### ISSUE TITLE
Make tabs "sticky"

### ISSUE THREAD

**Issue by danicahelb** [VEuPathDB/web-eda#1518]
when the user scrolls down a page, for example the Study details tab, the tabs should always remain visible instead of scrolling out of view. We want the user to be able to "save their place" on the study details tab (for ex, a description of the study arms) so they can come back and refer to it as they are working on their analysis.

currently, the tabs scroll out of view so the user must navigate away from their position on the study details tab in order to navigate to the B&S tab

**Comment by bobular** [comment]
Isn't this already implemented somehow? I sometimes see it, and other times I don't. But I've never explored why/how.

**Comment by moontrip** [comment]
I didn't look at this carefully (will do that today) but my suspect is that unlike local dev site, the production site has top logo/banner/menu, so perhaps a sort of interference occurs.

**Comment by dmfalke** [comment]
That's right, @moontrip. They are sticky, but on the ClinEpiDB website, the header is over them. We might have to add a prop to specify the offset to account for various headers.
